components/invitation_modal/invitation_modal_members_step/invitation_modal_members_step.jsx

```diff
@@ -270,7 +293,7 @@ class InvitationModalMembersStep extends React.PureComponent {
                         <UsersEmailsInput
                             usersLoader={this.usersLoader}
                             placeholder={placeholder}
-                            showError={this.shouldShowPickerError()}
+                            showError={errorProperties.showError}
```

Comment:
You could simplify this + the 3 props below with `{...errorProperties}` since they keys match the prop names.

Reply:
True that! Good catch. Thanks

Reply:
Done!